src/modules/transaction/components/TransactionDetailRow/TransactionDetailRow.test.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { screen, render, fireEvent, waitFor } from '@testing-library/react';
+import { truncateAddress } from 'src/modules/wallet/utils/account';
+import TransactionDetailRow from './TransactionDetailRow';
+import { mockTransactions } from '../../__fixtures__';
+
+jest.mock('react-i18next', () => ({
+  ...jest.requireActual('react-i18next'),
+  useTranslation: jest.fn().mockReturnValue({ t: jest.fn((val) => val) }),
+}));
+
+describe('TransactionDetailRow', () => {
+  const props = {
+    data: {
+      tooltip: 'test-tooltip',
+      value: 'test value',
+      label: 'test-title',
+    },
+    isParamsCollasped: false,
```

Comment:
@eniolam1000752 Please fix typo here because there's a mismatch between the props for the test and the props for the component